components/chef-automate-collect/commands/metadata_collectors.go

```diff
@@ -0,0 +1,240 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	fpath "path/filepath"
+	"strings"
+
+	req "github.com/chef/automate/api/external/cfgmgmt/request"
+	c "github.com/chef/automate/lib/platform/command"
+)
+
+var SCMType_UNKNOWN_SCM = req.SCMType_name[int32(req.SCMType_UNKNOWN_SCM)]
+var SCMType_GIT = req.SCMType_name[int32(req.SCMType_GIT)]
+var SCMWebType_UNKNOWN_SCM_WEB = req.SCMWebType_name[int32(req.SCMWebType_UNKNOWN_SCM_WEB)]
+var SCMWebType_GITHUB = req.SCMWebType_name[int32(req.SCMWebType_GITHUB)]
+
+type RolloutMetadata struct {
+	policyLockPath   string
+	PolicyName       string `json:"policy_name"`
+	PolicyNodeGroup  string `json:"policy_node_group"`
+	PolicyRevisionID string `json:"policy_revision_id"`
+	PolicyDomainURL  string `json:"policy_domain_url"`
+	Description      string `json:"description"`
+	SCMMetadata
+	CiMetadata
+}
+
+type SCMMetadata struct {
+	policyLockPath         string
+	SCMType                string `json:"scm_type"`
+	SCMWebType             string `json:"scm_web_type"`
```

Comment:
I think SCM-related fields are changing to not be explicitly "SCM",  but it might be we need to re-sync after Dan has completed the related WIP.

Reply:
Correct, we'll need to re-sync this after those changes. There is some future work we need to do around getting integration tests setup so we can see when the API change causes failures.